src/server/logs/logs.go

```diff
@@ -519,30 +530,67 @@ type passFunc func(labels map[string]string, msg *logs.LogMessage) bool
 // An adapter publishes log entries to a ResponsePublisher in a specified format.
 type adapter struct {
 	responsePublisher ResponsePublisher
+	level             logs.LogLevel
 	// If pass returns false, the message will not be published.
 	pass passFunc
 }
 
+func (a *adapter) addLevelFilter(l logs.LogLevel) {
+	if l > logs.LogLevel_LOG_LEVEL_UNSET && a.level == logs.LogLevel_LOG_LEVEL_UNSET {
```

Comment:
I think that `l != logs.LogLevel_LOG_LEVEL_UNSET` would express the intent better.

Reply:
Done.